lib/susedistribution.pm

```diff
@@ -96,14 +96,17 @@ sub x11_start_program($$$) {
         send_key('alt-t');
         sleep 3;
     }
-    send_key('ret', 1);
+    send_key "ret";
+    wait_still_screen;
     # make sure desktop runner executed and closed when have had valid value
```

Comment:
Sometimes due to high load I guess, keyboard shortcut misses it's chance, so we better try it more than once.
I don't know why is there this "valid" condition, what is the case for it ... :grey_question: